roles/openshift_common/tasks/main.yml

```diff
@@ -21,3 +21,4 @@
 
 - name: Set hostname
   hostname: name={{ openshift.common.hostname }}
+  when: not is_containerized | bool
```

Comment:
We should remove this all together, this workaround is no longer needed. We may need to verify against the first 3.0 builds just to be sure though.